cg/services/analysis_starter/configurator/implementations/raredisease.py

```diff
@@ -2,9 +2,14 @@
 from pathlib import Path
 
 from cg.constants import FileExtensions, Priority, Workflow
+from cg.io.txt import concat_txt
 from cg.models.cg_config import RarediseaseConfig
 from cg.services.analysis_starter.configurator.abstract_service import Configurator
 from cg.services.analysis_starter.configurator.models.nextflow import NextflowCaseConfig
+from cg.services.analysis_starter.configurator.utils import (
```

Comment:
Im starting to consider that the configurator class may grow too big. What would you guys think of creating sub services for building the nextflow config and the params?